python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -727,10 +735,7 @@ def __get_app_store_proxy_setting(self):
         try:
             config_data = shotgun.get_associated_sg_config_data()
         except UnresolvableCoreConfigurationError:
-            # This core is not part of a pipeline configuration, we're probably bootstrapping,
-            # so skip the check and simply return the regular proxy settings.
-            log.debug("No core configuration was found, using the current connection's proxy setting.")
-            return self._sg_connection.config.raw_http_proxy
+            config_data = None
```

Comment:
This reasoning was bad. We're looking to get a proxy for the app store, so using the one from `_sg_connection` without looking at the user settings is a mistake.